SensorFusion_src/suppoort_degree_matrix.c

```diff
@@ -0,0 +1,32 @@
+/******************************************************
+* Author : Shyam Bhuptani 
+* Function : Support Degree Matrix 
+* Application : This function converts the input array[N] (the array of various 
+* sensor readings) into a SDM matrix of which is having degre N*N
+* Input : size of sensor readings array N as integer and sensor_readings[N] of doubles 
+* Output : N*N support degree matrix
+*/
+
+/*Including dependancies*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include "../include/support_degree_matrix.h"
+
+/*Function declaration which returns a 2D array */
+double ** support_degree_matrix(double sensor_readings[], int n)
```

Comment:
Same idea here. The function should describe the action of the function, e.g. **build**_sdm().

Reply:
Updated new name in recent commit, please review